doc/setup.md

```diff
@@ -180,6 +164,7 @@ If the command outputs information about a service, it means that port 5000 is a
 
 	```
 	sudo apt install supervisor postgresql \
+	libpq-dev python3-pip \
```

Comment:
Why this addition, did you need that to run it?

Reply:
I'm adding back a line I accidentally removed in a previous commit. The formatting changed a bit which is why it looks like an addition/change in the git diff.